presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -163,7 +163,7 @@ private static AWSGlueAsync createAsyncGlueClient(GlueHiveMetastoreConfig config
             asyncGlueClientBuilder.setRegion(config.getGlueRegion().get());
         }
         else if (config.getPinGlueClientToCurrentRegion()) {
-            Region currentRegion = Regions.getCurrentRegion();
+            Region currentRegion = getCurrentRegionFromEC2Metadata();
```

Comment:
`getCurrentRegionFromEC2Metadata()` isn't more informative than `getCurrentRegion()` or `Regions. getCurrentRegion()`.

Perhaps rename it to `getRegionCached()` or `RegionCache.getRegion()` (without static import)